qt/python/mantidqt/widgets/test/test_workspacetreewidget.py

```diff
@@ -0,0 +1,45 @@
+#  This file is part of the mantid workbench.
+#
+#  Copyright (C) 2017 mantidproject
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+from __future__ import absolute_import, print_function
+
+import unittest
+
+from qtpy.QtWidgets import QApplication
+
+from mantidqt.widgets.workspacewidget.mantidtreemodel import MantidTreeModel
+from mantidqt.widgets.workspacewidget.workspacetreewidget import WorkspaceTreeWidget
+
+QAPP = None
+
+
```

Comment:
We have created a decorator to simplify tests that require a running QApplication so that we don't need to duplicate the `setupClass` stuff over and over: See [here](mantidproject/mantid/blob/5e0544dcdcf9e47149c67c2d0d3bfd4f102a6d5d/qt/applications/workbench/workbench/plugins/test/test_jupyterconsole.py#L30) for an example.